pkg/convert/pot.go

```diff
@@ -0,0 +1,97 @@
+package convert
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"strings"
+
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/pot"
+)
+
+func ToPot(m model.Messages) ([]byte, error) {
+	var b bytes.Buffer
+
+	b.WriteString(fmt.Sprintf("\"Language: %s\n", m.Language))
```

Comment:
check err

Reply:
inconsistent with other printing to buffer